recipe/recipe.yaml

```
@@ -0,0 +1,57 @@
+schema_version: 1
+
+context:
+  name: periodiq
+  version: 0.13.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/periodiq-${{ version }}.tar.gz
+  sha256: ecd83d7614bb9e019aa2718b63c4ffe0a6c77fa0d49f33fd90c309982ddcf7cb
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  python:
+    entry_points:
+      - periodiq = periodiq:entrypoint
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - poetry >=0.12
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - dramatiq >=1.5.0,<2.0.0
+    - pendulum >=2.0,<4
+    # try setuptools here for pkg_resources
+    - setuptools
```

This comment could be improved; if periodiq is using `pkg_resources`, it's not about _trying_: you _need_ setuptools. Also, you need a cap on the version, because setuptool has been emitting a warning:
```
UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
```

```suggestion
    # package still uses pkg_resources, which is deprecated in setuptools though
    - setuptools <81
```